### DISCUSSION TITLE
Has this repo been abandoned by c't?

### DISCUSSION THREAD

**Discussion by 8xiom** [ct-Open-Source/ct-Smart-Home#79] [Q&A]
Hi all,
It seems that there is no more activity on this repo. In particalur the automatic deployment of the docker image seems to have stopped. I figure that Merlin is no longer with c't and nobody has stepped in so far. That's a pity since this setup works very well on my Raspi. It's the best solution for me that I've come across so far.
So to the people out there at c't: Could you provide us users with some information regarding the future of the ct-Smart-Home?
Thanks a lot!

**Comment by fischkai** [comment]
My smart home depends on this. It controls a lot of devices. Would be nice to know about the future of this project.
